test/fixtures/output/asyncapi/import-with-overwrite.golden

```diff
@@ -1,2 +1,4 @@
-Schema registered under subject topic1-value with ID 1.
+Schema registered under subject "topic1-value" with ID "1".
+Tags added to schema "1".
```

Comment:
Can we list the tags here?
```
Tags "2", "3", and "4" added to schema...
```

Reply:
I have added it as an array of strings. 
e.g.: Tags [2 3 4] added to schema...

Reply:
See my other comment, this is a bit lazy and we can make the output nicer here.

Reply:
Modified it using ArrayToCommaDelimitedString() function